File: benchmark-java/src/main/java/benchmark/java/metrics/Info.java
package benchmark.java.metrics;

import benchmark.java.Config;


/**
 *
 * @author Jan
 */
public class Info {
	
	private Config.Format format;
	private String name;
	private String url;
	private String version;

	public Info(Config.Format format, String name, String url, String version) {
		this.format = format;
		this.name = name;
		this.url = url;
		this.version = version;
	}

	public Info(Config.Format format, String name, String url) {
		this.format = format;
		this.name = name;
		this.url = url;
	}
	

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}

	public String getVersion() {
		return version;
	}

	public void setVersion(String version) {
		this.version = version;
	}

	public Config.Format getFormat() {
		return format;
	}

	public void setFormat(Config.Format format) {
		this.format = format;
	}

	public String getFullName() {
		if (version == null || "".equals(version)){
			return format + " - " + name;
		}
		return format + " - " + name + " " + version;
	}
	
	
}

File: benchmark-php/app/Metrics/Info.php
<?php
/**
 * Created by PhpStorm.
 * User: Jan
 * Date: 14. 2. 2017
 * Time: 13:18
 */

namespace Benchmark\Metrics;


use Benchmark\Config;

class Info
{
	/** @var string */
	private $format;

	/** @var string */
	private $name;

	/** @var string */
	private $url;

	/** @var string */
	private $version;

	/**
	 * Info constructor.
	 * @param string $format
	 * @param string $name
	 * @param string $url
	 * @param string $version
	 */
	public function __construct($format, $name, $url, $version = NULL)
	{
		$this->format = $format;
		$this->name = $name;
		$this->url = $url;
		$this->version = $version;
	}

	/**
	 * @return string
	 */
	public function getFullName()
	{
		if (!$this->version){
			return 'php - ' . $this->format . ' - ' . $this->name;
		}
		return 'php - ' . $this->format . ' - ' . $this->name . ' ' . $this->version;
	}

	/**
	 * @return string
	 */
	public function getFormat()
	{
		return $this->format;
	}

	/**
	 * @param string $format
	 */
	public function setFormat($format)
	{
		if (in_array($format, Config::FORMATS)) {
			$this->format = $format;
		}
	}

	/**
	 * @return string
	 */
	public function getName()
	{
		return $this->name;
	}

	/**
	 * @param string $name
	 */
	public function setName($name)
	{
		$this->name = $name;
	}

	/**
	 * @return string
	 */
	public function getUrl()
	{
		return $this->url;
	}

	/**
	 * @param string $url
	 */
	public function setUrl($url)
	{
		$this->url = $url;
	}

	/**
	 * @return string
	 */
	public function getVersion()
	{
		return $this->version;
	}

	/**
	 * @param string $version
	 */
	public function setVersion($version)
	{
		$this->version = $version;
	}


}